src/truefoundry_sdk/types/aws_sagemaker_provider_account_auth_data.py

```
@@ -3,6 +3,6 @@
 import typing
 
 from .aws_access_key_based_auth import AwsAccessKeyBasedAuth
-from .aws_bedrock_assumed_role_based_auth import AwsBedrockAssumedRoleBasedAuth
+from .sagemaker_assumed_role_based_auth import SagemakerAssumedRoleBasedAuth
 
-AwsSagemakerProviderAccountAuthData = typing.Union[AwsAccessKeyBasedAuth, AwsBedrockAssumedRoleBasedAuth]
+AwsSagemakerProviderAccountAuthData = typing.Union[AwsAccessKeyBasedAuth, SagemakerAssumedRoleBasedAuth]
```

Ruff (0.15.9)

[warning] Use `X | Y` for type annotations

(UP007)